src/index.html

```diff
@@ -11,6 +11,31 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="GET"
+      class="search-panel__big"
+      data-qa="big"
+    >
+      <input
+        data-qa="keypress"
+        class="search-bar search-bar--big"
```

Comment:
If you add all styles for big input (base input) to the search-bar class and change style for small input, you don't need search-bar--big class anymore. Look at 2 point on [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)